dist/api_data.py

```diff
@@ -1081,6 +1081,9 @@ def __ge__(self, other):
         Config('hashsize', '1024', r'''
             number of buckets in the hashtable that keeps track of objects''',
             min='64', max='1048576'),
+        Config('flushed_data_cache_insertion', 'true', r'''
+            configure enabling caching of freshly-flushed data, before it is removed locally.''',
```

Comment:
NP:  'enable caching of freshly-flush...' would suffice so we don't mix tenses.

Reply:
Update [473f1d9](https://github.com/wiredtiger/wiredtiger/pull/9650/commits/473f1d9dcbc2223f3b1ebb8d49c8034b57845e08)